.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+- [ ] Closes #xxx
+- [ ] Code changes are covered by tests
+- [ ] New functions added to `__init__.py`
+- [ ] API.rst is up to date, along with other sphinx docs pages
+- [ ] Example notebooks are rerun and differences in results scrutinized
+- [ ] Updated changelog
```

Comment:
Maybe add items like these from pvlib?
- [ ] Pull request is nearly complete and ready for detailed review.
- [ ] Maintainer: Appropriate GitHub Labels and Milestone are assigned to the Pull Request and linked Issue.

Reply:
I think the first is covered by github's draft functionality so we can skip that. I could go either way on the "maintainer" item.